### ISSUE TITLE
Information on the linguistic structures in 2,679 languages

### ISSUE THREAD

**Issue by gkaramanis** [rfordatascience/tidytuesday#77]
[](https://www.kaggle.com/rtatman/world-atlas-of-language-structures)

> There are over 7,000 human languages in the world. The World Atlas of Language Structures (WALS) contains information on the structure of 2,679 of them. It also includes information about where languages are used. WALS is widely-cited and used in the linguistics research community.

**Comment by jonthegeek** [comment]
Unfortunately there doesn't seem to be a good article demonstrating this dataset. I love language, though, so maybe I'll do something with this to make it more usable in the future!

**Comment by tracykteal** [comment]
@jonthegeek would you still want to do something with this dataset?

**Comment by jonthegeek** [comment]
@tracykteal Yes, thanks for the reminder! The dataset on Kaggle probably has enough of a description to "count" as the article half, so I think we're fine to use this!

**Comment by darakhshannehal** [comment]
Hi @jonthegeek, I think this dataset suggestion is similar to the one in #908, for which I submitted a PR. If that PR got merged, maybe both issues can be closed?

**Comment by jonthegeek** [comment]
@darakhshannehal Agreed! I'll go ahead and mark this one as (close enough to) a duplicate. Thanks!
